app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -37,6 +37,7 @@ CONFIG_INTEL_ADSP_TIMER=y
 CONFIG_MM_DRV_INTEL_ADSP_TLB_REMAP_UNUSED_RAM=y
 CONFIG_AMS=y
 CONFIG_COUNTER=y
+CONFIG_TELEMETRY=y
 
```

Comment:
Configs should be enabled in a separate patch.

Reply:
Separate commit or separate PR?

Reply:
patch = commit.

The lines between commits is very rarely ambiguous. The line between PRs is less well-defined.

Drawing the lines between commits is easy: just think about `git cherry-pick`, `git revert`, `git bisect` etc. There are many pages about this, this is my favorite: https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes

Drawing the line between PRs is more difficult. (1) put yourself in the shoes of reviewers (2) think "Continuous Integration".

In this case: would it be good to have one or two round of daily tests with the code merged but disabled?

This is the most relevant page:
https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#defining-smaller-prs